airbyte-integrations/connectors/source-monday/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY source_monday ./source_monday
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.6
+LABEL io.airbyte.version=0.2.7
```

Comment:
this should be a major version bump - 1.0.0
https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors

Reply:
Fixed